libr/core/cmd.c

```diff
@@ -1266,6 +1282,239 @@ static int cmd_stdin(void *data, const char *input) {
 	return r_core_run_script (core, "-");
 }
 
+static void load_table(RCore *core, RTable *t, const char *file) {
+	char *data = NULL;
+
+	if (*file == '$') {
+		const char *cdata = r_cmd_alias_get (core->rcmd, file, 1);
+		if (cdata) {
+			data = strdup (cdata + 1);
+		}
+	} else {
+		data = r_file_slurp (file, NULL);
+	}
+	if (!data) {
+		eprintf ("Error: Cannot slurp '%s'\n", file);
+		return;
+	}
+	RListIter *iter;
+	char *line;
+	RList *lines = r_str_split_list (data, "\n", 0);
+	bool expect_header = false;
+	bool expect_rows = false;
+	const char *separator = "|";
+	int ncols = 0;
+	r_list_foreach (lines, iter, line) {
+		if (!expect_rows) {
+			if (r_str_startswith (line, ".--")) {
+				expect_header = true;
+				separator = "|";
+				continue;
+			}
+			if (r_str_startswith (line, "┌")) {
+				expect_header = true;
+				separator = "│";
+				continue;
+			}
+			if (r_str_startswith (line, ")-")) {
+				expect_rows = true;
+				separator = "|";
+				continue;
+			}
+			if (r_str_startswith (line, "│─")) {
+				expect_rows = true;
+				separator = "│";
+				continue;
+			}
+		}
+
+		RTableColumnType *typeString = r_table_type ("string");
+		RTableColumnType *typeNumber = r_table_type ("number");
+		if (expect_header) {
+			char *arg;
+			RList *args = r_str_split_list (line + strlen (separator), separator, 0);
+			RListIter *iter2;
+			ncols = 0;
+			if (r_list_length (t->cols) > 0) {
+				eprintf ("Warning: Not re-adding headers. Use ,- to reset the table.\n");
+				continue;
+			}
+			r_list_foreach (args, iter2, arg) {
+				char *s = strchr (arg, ' ');
```

Comment:
why is `strchr` needed here?

Reply:
to skip the first word

Reply:
why do you want to skip it?
```
.-------------------------.
| foo fuffa | bar  | cow  |
)-------------------------(
| ciao      | zoo  | boh  |
`-------------------------'
```
By doing so, if you load the above table `foo fuffa` becomes `fuffa`. Is that done on purpose?